kroto-plus-coroutines/src/test/kotlin/com/github/marcoferrer/krotoplus/coroutines/client/ClientCallBidiStreamingTests.kt

```diff
@@ -332,4 +343,58 @@ class ClientCallBidiStreamingTests {
         assert(responseChannel.isClosedForReceive) { "Response channel should be closed for receive" }
     }
 
+    @Test
+    fun `High throughput call succeeds`() {
```

Comment:
This test was introduced as part of the discussion the was had with @chris-blacker in https://github.com/marcoferrer/kroto-plus/pull/59